vllm/v1/worker/kv_connector_model_runner_mixin.py

```diff
@@ -138,17 +137,17 @@ def _get_kv_connector_output(
             )
             output.invalid_block_ids = kv_connector.get_block_ids_with_load_errors()
 
-            output.kv_connector_stats = (
-                KVConnectorModelRunnerMixin.get_kv_connector_stats()
-            )
+            if has_kv_transfer_group():
+                output.kv_connector_stats = (
+                    get_kv_transfer_group().get_kv_connector_stats()
```

Comment:
I think you can replace `get_kv_transfer_group()` with `kv_connector` here and down below.

Reply:
Good suggestion. Done.